src/multibody/liegroup/liegroup-base.hpp

```diff
@@ -356,11 +356,26 @@ PINOCCHIO_LIE_GROUP_PUBLIC_INTERFACE_GENERIC(Derived,typename)
      * @brief      Normalize the joint configuration given as input.
      *             For instance, the quaternion must be unitary.
      *
-     * @param[out]     qout  the normalized joint configuration.
+     * @note       If the input vector is too small (i.e., qout.norm()==0), then it is left unchanged.
```

Comment:
This is not really what it means too small. Being more precise would help.

Reply:
This is basically a copy-paste from Eigen's own documentation. Can't be safer and more precise than this

Reply:
Do know under which precision they consider really a zero to be zero?

Reply:
Absolutely no idea. Again, since we decided to accept Eigen's behavior, I just blindly copy-pasted that part of the doc, you can check it [here](https://eigen.tuxfamily.org/dox/classEigen_1_1MatrixBase.html#ad16303c47ba36f7a41ea264cb26bceb6). I agree it is very weird, I was surprised myself, in computer science you never say that a floating-point number _is_ zero, but that's verbatim from Eigen's doc and it's the best we have.

I suppose we _could_ dig into Eigen's code and find out, but it would be a mistake to document in Pinocchio's doc something which is not documented by Eigen itself and is therefore likely to change with no guarantees for us nor for the users.

Reply:
Resolving this conversation with no changes